src/Mgate/PersonneBundle/Resources/views/Employe/ajouter.html.twig

```diff
@@ -8,33 +8,84 @@
     {% endblock %}
 {% else %}
 
-
 {% endif %}
     #}
+
 {% block title %}
-    Ajouter un employé à {{ prospect.nom }} - {{ parent() }}
+    {{ 'employee.add_employee'|trans({}, 'person') }} {{ parent() }}
 {% endblock %}
 
 {% block content_title %}
-    Ajouter un employé à {{ prospect.nom }}
+    {{ 'employee.add_employee_to'|trans({}, 'person') }} {{ prospect.nom }}
 {% endblock %}
 
 {% block breadcrumb %}
     <ul class="breadcrumb no-margin pull-right">
-        <li>
-            <a href="{{ path('MgatePersonne_prospect_homepage') }}">Liste des prospects</a>
-        </li>
-        <li>
-            <a href="{{ path('MgatePersonne_prospect_voir', {'id': prospect.id}) }}">{{ prospect.nom }}</a>
-        </li>
-        <li>
-            Ajouter un employé
-        </li>
+        <li><a href="{{ path('MgatePersonne_prospect_homepage') }}">{{ 'prospect.prospect_list'|trans({}, 'person') }}</a></li>
+        <li><a href="{{ path('MgatePersonne_prospect_voir', {'id': prospect.id}) }}">{{ prospect.nom }}</a></li>
+        <li class="active">{{ 'employee.add_employee'|trans({}, 'person') }}</li>
     </ul>
 {% endblock %}
 
 {% block content_bundle %}
-
-    {% include "MgatePersonneBundle:Employe:formulaire.html.twig" %}
-
+    <div class="table-responsive content">
```

Comment:
Because the last table line between the two views are slightly different. Should I move only the common rows to this separate file ?